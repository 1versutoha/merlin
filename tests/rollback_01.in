["tell","struct","let x = prerr_endline \"LOL\"\n\n"]
["tell","end",null]
["errors"]
["seek","before",{"line":3,"col":0}]
["tell","struct","\n\n  ;\n  prerr_endline \"TROLOLO\"\n"]
["tell","end",null]
["errors"]
["seek","before",{"line":5,"col":0}]
["tell","struct","\n\nand y = 7\n"]
["tell","end",null]
["errors"]
