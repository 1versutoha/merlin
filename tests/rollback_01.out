["return",false]
["return",true]
["return",[]]
["return",{"line":1,"col":27}]
["return",false]
["return",true]
["return",[]]
["return",{"line":4,"col":25}]
["return",false]
["return",true]
["return",[]]
